cmd/svcat/main.go

```diff
@@ -48,6 +50,12 @@ func main() {
 }
 
 func buildRootCommand() *cobra.Command {
+	// Make cobra aware of select glog flags
+	// Enabling all flags causes unwanted deprecation warnings from glog to always print in plugin mode
+	pflag.CommandLine.AddGoFlag(flag.CommandLine.Lookup("v"))
+	pflag.CommandLine.AddGoFlag(flag.CommandLine.Lookup("logtostderr"))
+	pflag.CommandLine.Set("logtostderr", "true")
```

Comment:
Uh, doesn't this force it to log to std err always? Do we not allow redirection of that?

Reply:
Without that flag, I don't see any verbose output. Redirection seems to work fine though

```
$ kubectl plugin svcat get brokers -v=7 2> debug.log
     NAME                                  URL                                STATUS
+------------+--------------------------------------------------------------+--------+
  osba         http://osba-open-service-broker-azure.osba.svc.cluster.local   Ready
  ups-broker   http://ups-broker-ups-broker.ups-broker.svc.cluster.local      Ready


$ cat debug.log
I0308 12:18:48.735569   27194 loader.go:357] Config loaded from file /Users/carolynvs/.kube/config
I0308 12:18:48.792845   27195 loader.go:357] Config loaded from file /Users/carolynvs/.kube/config
I0308 12:18:48.793937   27195 round_trippers.go:414] GET https://192.168.99.100:8443/apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers
I0308 12:18:48.793955   27195 round_trippers.go:421] Request Headers:
I0308 12:18:48.793965   27195 round_trippers.go:424]     Accept: application/json, */*
I0308 12:18:48.793974   27195 round_trippers.go:424]     User-Agent: svcat/v0.0.0 (darwin/amd64) kubernetes/$Format
I0308 12:18:48.806120   27195 round_trippers.go:439] Response Status: 200 OK in 12 milliseconds
```

Reply:
Since this is a cli, and that flag says "log to the console instead of a file", it seems appropriate to have it always on.

Reply:
Huh, my confusion was that it seems like you're adding a flag to control it in line 6, but then always setting it to true on line 7. If it's always on why add the flag on line 6?

Reply:
Lines 6 adds the flag to the flag set so that it is defined, has helptext, etc. Line 7 sets it to true. Glog directly reads the flag value, logtostderr isn't something that I can directly control myself, so I have to muck with the flag...

Reply:
so does this line force glog to log warnings to stderr? I'm not familiar enough with pflag or glog to know this offhand, so I'm hoping you can save me the research 😄